### ISSUE TITLE
[Accessibility]A11y_VSCode_ViewProblems_ScreenReader: VoiceOver reads state of each tab item as Selected.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#52046]
**Environment Details:** 
VSCode Version : 1.24.0
OS Version : Win10

**Additional Details:**
MAS Violated: MAS1.3.1
ScreenReader Version: HighSierra10.13.5 

Repro Steps:
1. Launch VS Code while NVDA is ON.
2. Go to View menu and Select Problems.
3. Check the state of all the tabs(Problems,Output,Debug Console, Terminal).

Actual:
VoiceOver reads the state of each tab item (Problems,Output,Debug Console, Terminal) as "Selected" whether it is selected or non selected.

Expected:
Voiceover should read the selected tab item as selected and non selected tab item as non selected.

Recommendations:
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

User Impact:
User will not understand which tab item is selected and which one is non selected.

MAS Reference:
https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

Attachment for reference: 
[A11y VSCode Problems Voiceover reads selected.pptx](https://github.com/Microsoft/vscode/files/2106182/A11y.VSCode.Problems.Voiceover.reads.selected.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by isidorn** [comment]
Dup of https://github.com/Microsoft/vscode/issues/51967
